phpunit/class-wp-rest-post-format-search-handler-test.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * WP_REST_Post_Format_Search_Handler tests
+ *
+ * @package Gutenberg
+ */
+
+/**
+ * Tests for WP_REST_Post_Format_Search_Handler_Test.
+ *
+ * @group restapi
+ */
+class WP_REST_Post_Format_Search_Handler_Test extends WP_Test_REST_Controller_Testcase {
+
+	/**
+	 * Term associated with a post format.
+	 *
+	 * @var array
+	 */
+	private static $my_post_format_term = array();
+
+	/**
+	 * Create fake data before our tests run.
+	 *
+	 * @param WP_UnitTest_Factory $factory Helper that lets us create fake data.
+	 */
+	public static function wpSetUpBeforeClass( $factory ) {
+		self::$my_post_format_term = $factory->term->create(
+			array(
+				'taxonomy' => 'post_format',
+				'name'     => 'Aside',
+				'slug'     => 'post-format-aside',
+			)
+		);
+	}
+
+	/**
+	 * Delete our fake data after our tests run.
+	 */
+	public static function wpTearDownAfterClass() {
+		wp_delete_term( self::$my_post_format_term, true );
+	}
+
+	/**
+	 * Search through terms of any type.
+	 */
+	public function test_get_items_search_type_post_format() {
+		$response = $this->do_request_with_params(
+			array(
+				'per_page' => 100,
+				'type'     => 'post-format',
+			)
+		);
+		$this->assertEquals( 200, $response->get_status() );
+		$this->assertEqualSets(
```

Comment:
What about checking if the results contains "Aside"? This feels brittle, I think it would fail if there are other post formats created.

Reply:
Done in 26e1cb0173.